pkg/atlantis/README.md

```diff
@@ -0,0 +1,146 @@
+# atlantis integration
+
+## Dynamic Repo Config Generation
+
+If you want to generate `atlantis.yaml` before Atlantis can parse it,
+you can add the following run command to [pre_workflow_hooks](https://www.runatlantis.io/docs/pre-workflow-hooks.html#pre-workflow-hooks).
+The `atlantis.yaml` repo config file will be generated right before Atlantis will parse it.
+
+```yaml
+repos:
+  - id: /.*/
+    pre_workflow_hooks:
+      - run: ./repo-config-generator.sh
```

Comment:
Instead of the shell script, couldn't you do something in-line like this?

```suggestion
      - run: |
          atmos terraform generate varfiles --file-template=varfiles/{tenant}-{environment}-{stage}-{component}.tfvars.json
          atmos atlantis generate repo-config --config-template config-template-1 --project-template project-template-1 --workflow-template workflow-template-1
        
```

Reply:
Lets do inline